editors/vscode/syntaxes/jakt.tmLanguage.json

```diff
@@ -784,8 +784,16 @@
           "match": "\\b(as)\\b\\s"
         },
         {
-          "name": "keyword.control.other.jakt",
-          "match": "\\b(import)\\b"
+          "name": "keyword.control.import.jakt",
+          "match": "\\b(import)(?:\\s*)(?:(?<=\\s+)(?:(extern\\s+c)|(extern)))?\\b",
+          "captures": {
+            "2": {
+              "name": "keyword.control.import.extern-c.jakt"
```

Comment:
:thinking: that's a valid scope name? I thought they had to match `(\w+\.)+\w+`

Reply:
There is already scopes like `fat-arrow` and `return-throws`. If that is true we should fix all of them

Reply:
Oh, if it works then it's alright, it's just a name